docs/getting_started.md

```diff
@@ -132,36 +132,27 @@ Experiment with the graph range parameters and other settings.
 Let us make this more interesting and start some example targets for Prometheus
 to scrape.
 
-The Go client library includes an example which exports fictional RPC latencies
-for three services with different latency distributions.
-
-Ensure you have the [Go compiler installed](https://golang.org/doc/install) and
-have a [working Go build environment](https://golang.org/doc/code.html) (with
-correct `GOPATH`) set up.
-
-Download the Go client library for Prometheus and run three of these example
-processes:
+The next steps are for `node_exporter` to install follow
```

Comment:
This doesn't parse. I'd go for something like:

The node exporter is used as the example target, for more on using it see...